app/components/application_component.rb

```diff
@@ -1,3 +1,3 @@
 class ApplicationComponent < ViewComponent::Base
-
+  delegate :can?, to: :helpers
```

Comment:
Before when we were delegating current_user,. we just did it specifically on each component that needed it, instead of on ApplciationComponent. Have to write it each time, but it's more clear where it's coming from, right where you're using it.  Maybe stick with putting it on each component, what do you think?

Reply:
Oh, I can't get it to work; raises an `undefined method` error. Sort of annoying since several of the subclasses are delegating other methods to `:helpers`. For now I'm just leaving this as is with a comment.